Wikipedia/Code/Session.swift

```diff
@@ -200,8 +200,9 @@ import Foundation
         }
         switch httpResponse.statusCode {
         case 401:
-            WMFAuthenticationManager.sharedInstance.logout {
+            WMFAuthenticationManager.sharedInstance.logout(initiatedBy: .server) {
                 self.removeAllCookies()
+                NotificationCenter.default.post(name: Session.wasDeauthenticatedNotification, object: nil)
```

Comment:
Should this notification move into the authentication manager so it's also posted when logout is initiated by `.app`?